src/dmd/frontend.h

```diff
@@ -6660,6 +6662,7 @@ struct Target
 public:
     void _init(const Param& params);
     void setCPU();
+    void setTriple(Triple triple);
```

Comment:
Seems like dtoh is generating invalid code here @MoonlightSentinel - struct passed by value, but we only have an opaque declaration.